pom.xml

```diff
@@ -1629,17 +1629,21 @@
     </profile>
 
     <profile>
-      <id>hadoop-2.2</id>
+      <id>hadoop-1</id>
       <properties>
-        <hadoop.version>2.2.0</hadoop.version>
-        <protobuf.version>2.5.0</protobuf.version>
-        <hbase.version>0.98.7-hadoop2</hbase.version>
-        <avro.mapred.classifier>hadoop2</avro.mapred.classifier>
-        <codehaus.jackson.version>1.9.13</codehaus.jackson.version>
+        <hadoop.version>1.2.0</hadoop.version>
```

Comment:
None of this should actually affect the releases people use since they build with profiles explicitly enabled as desired. It's a good change of defaults, for hygiene though.

The story here is that Spark 1.2 was sorta accidentally released as depending (in Maven) on 2.2, and so that was expressed later, but this is the logical conclusion of that process. The default build which you'd get if approaching this as a developer is and should be a consistent Hadoop 2.2 build.

However more needs to change besides this. The release script has to set a "hadoop-1" profile in some cases now. The docs about building vs various Hadoop versions has to be updated too.